handbook/product/beta_and_experimental_feature_labels.md

```diff
@@ -0,0 +1,61 @@
+# Beta and experimental feature labels
+
+We use a hierarchy of feature labels to set expectations with our users and internally: `Experimental` << `Beta` << GA. 
+
+GA ("General Availability") has no in-product label and is the default user assumption for an unlabelled feature. 
```

Comment:
Do we have a better (internal maybe?) resource to link to what GA means?

```suggestion
[GA ("General Availability")](https://www.productplan.com/glossary/general-availability/) has no in-product label and is the default user assumption for an unlabelled feature. 
```

Reply:
I don't think so, at least not yet. Will add to my backlog of possible docs to add and go with this for now.